modules/git/log_name_status.go

```diff
@@ -354,7 +356,7 @@ heaploop:
 		}
 		current, err := g.Next(treepath, path2idx, changed, maxpathlen)
 		if err != nil {
-			if err == context.DeadlineExceeded {
+			if err == context.DeadlineExceeded || errors.Unwrap(err) == context.DeadlineExceeded {
```

Comment:
Isn't `error.Is` made for this purpose?

Reply:
We have been burnt several times before with errors.Is - in this case - after having looked carefully at the code it should be correct.